packages/main/src/Button.js

```diff
@@ -169,6 +169,8 @@ const metadata = {
 		},
 
 		/**
+		 * Use aria-expanded to indicate the state of a collapsible element.
```

Comment:
same as the link, but I think that i you modify code in other component you should rename the PR name

Reply:
Done